au4/src/projectscene/qml/Audacity/ProjectScene/clipsview/ClipHandles.qml

```diff
@@ -346,6 +362,10 @@ Item {
         }
     }
 
+    Component.onCompleted: {
+        canvas = findParentWithObjectName(this, "clipsView")
```

Comment:
Let's also abandon searching for a parent by name and pass it directly through all the children?
I don't like the search approach because it's hard to control. Someone will delete or just rename the name and everything will break